cafebabel/templates/articles/drafts/create.html

```diff
@@ -1,11 +1,17 @@
 {% extends "base.html" %}
 
+{% block body_class %}create-article-page{% endblock %}
 {% block body %}
   <form method=post action={{ url_for('drafts.create') }} enctype=multipart/form-data>
-  <div class=text>
-    <label for=title>Title</label>
-    <input id=title name=title required>
+  <canvas></canvas>
```

Comment:
what's the problem with having `<canvas>` tag inside the template ?

Reply:
`<canvas>` right there has no content therefore no meaning.
It's an empty tag and reading the HTML, it's obvious we can remove it.

Also `<canvas>` could be used for other purposes in the project. But if we ever attempt to use it again, it will be decorated but the noise effect. Maybe we should at least apply a class on the canvas like `.noise` that we could use in the JS.

Reply:
yep